required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -1981,14 +1974,19 @@
   {
     \tag_mc_end_push:
 %    \end{macrocode}
-% test if a footnote is allowed, if not move up to the document structure.
+% test if a footnote is allowed, if not move up to the next sect or
+% the document structure.
```

Comment:
probably not as part of this PR but isn't that kind of somehting really common and we should have a command with some arguments that handle this situation?